### PR TITLE
Added release notes for #3406

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: docs/releasenotes/5.4.0.rst [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When accessing individual image pixels, negative indexes are now also accepted.
+For example, to get or set the farthest pixel in the lower right of an image:
+
+```
+px = im.load()
+print(px[-1, -1])
+px[-1, -1] = (0, 0, 0)
+```
```
Comment (hugovk) [inline docs/releasenotes/5.4.0.rst]: This is reStructuredText, not Markdown.

**Comment by hugovk** [comment]
Thanks, we should also add it to the regular docs.

Perhaps just in the example here? 

https://pillow.readthedocs.io/en/5.3.x/reference/PixelAccess.html#example

File: docs/releasenotes/5.4.0.rst (hunk 1 above)
Comment (radarhere) [reply docs/releasenotes/5.4.0.rst]: Okay, updated

**Comment by radarhere** [comment]
I've added it there, as well as https://pillow.readthedocs.io/en/5.3.x/reference/PyAccess.html#example

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**